tests/remote/test_thunderbird.py

```diff
@@ -4,248 +4,124 @@
-              'branch': 'comm-central',
-              'platform': 'linux64'}},
-
-    # -p mac --branch=comm-central
-    {'args': {'application': 'thunderbird',
-              'branch': 'comm-central',
-              'platform': 'mac'}},
-
-    # -p win32 --branch=comm-central --debug-build
-    {'args': {'application': 'thunderbird',
-              'branch': 'comm-central',
-              'platform': 'win32',
-              'debug_build': True}},
-
-    # -p win32 --branch=comm-central --locale=de
-    {'args': {'application': 'thunderbird',
-              'branch': 'comm-central',
-              'platform': 'win32',
-              'locale': 'de'}},
-
-    # -p linux --branch=comm-central --extension=txt
-    {'args': {'application': 'thunderbird',
-              'branch': 'comm-central',
-              'platform': 'linux',
-              'extension': 'txt'}},
-]
-
-
-class ThunderbirdRemoteTests(unittest.TestCase):
-    """Test all scraper classes for Thunderbird against the remote server"""
-
-    def setUp(self):
-        logging.basicConfig(format=' %(levelname)s | %(message)s', level=logging.ERROR)
-        self.logger = logging.getLogger(self.__class__.__name__)
-
-        # Create a temporary directory for potential downloads
-        self.temp_dir = tempfile.mkdtemp()
-
-    def tearDown(self):
-        mozfile.rmtree(self.temp_dir)
-
-    def test_release_scraper(self):
-        for test in tests_release_scraper:
-            scraper = mozdownload.ReleaseScraper(destination=self.temp_dir,
-                                                 logger=self.logger,
-                                                 **test['args'])
-            if test.get('url'):
-                self.assertEqual(urllib.unquote(scraper.url),
-                                 urljoin(BASE_URL, test['url']))
-
-    def test_candidate_scraper(self):
-        for test in tests_candidate_scraper:
-            scraper = mozdownload.ReleaseCandidateScraper(destination=self.temp_dir,
-                                                          logger=self.logger,
-                                                          **test['args'])
-            if test.get('url'):
-                self.assertEqual(urllib.unquote(scraper.url),
-                                 urljoin(BASE_URL, test['url']))
-
-    def test_daily_scraper(self):
-        for test in tests_daily_scraper:
-            mozdownload.DailyScraper(destination=self.temp_dir,
-                                     logger=self.logger,
-                                     **test['args'])
-
-    def test_tinderbox_scraper(self):
-        for test in tests_tinderbox_scraper:
-            mozdownload.TinderboxScraper(destination=self.temp_dir,
-                                         logger=self.logger,
-                                         **test['args'])
+@pytest.mark.parametrize("args,url", [
+    ({'application': 'thunderbird', 'platform': 'linux', 'version': '52.0'},
```

Comment:
If you could format the parameters similarly to the other ones below, that would be great.